setup.cfg

```diff
@@ -5,4 +5,4 @@ universal=1
 source=defopt
 
 [coverage:report]
-fail_under=98
+fail_under=97
```

Comment:
The `return` in
```
    if not isinstance(tag, str) and tag is not None:
        return
```
doesn't get exercised (and I don't think it's worth writing a test to specifically test it -- it comes straight from the upstream implementation).  That's enough for dropping the coverage.